source/features/clean-conversation-headers.tsx

```diff
@@ -23,7 +23,7 @@ function initPR(): void {
 	observe('.gh-header-meta .flex-auto', {
 		add(byline) {
 			const isMerged = select.exists('#partial-discussion-header [title="Status: Merged"]');
-			const isSameAuthor = select('.js-discussion > .TimelineItem:first-child .author')?.textContent === select('.author', byline)!.textContent;
+			const isSameAuthor = select('.js-discussion > .TimelineItem:first-child .author')?.textContent === select('.author', byline)?.textContent;
```

Comment:
What’s causing this issue exactly? How can I replicate it?

Reply:
Go to the file changes tab then go to the main tab

Reply:
Sorry commits tab

Reply:
I see. I don't think this is the right way to fix it though. Separately, this feature works on every tab, so something is changing only when switching tab and _that_ should probably be accounted for

Reply:
What I'm seeing is that this feature doesn't have `:not(.rgh-clean-conversation-header)` in the selector, so that might be it.

Reply:
Without looking I think the issue is that the author only exists in the conversation view. I will check later tonight

Reply:
> What I'm seeing is that this feature doesn't have `:not(.rgh-clean-conversation-header)` in the selector, so that might be it.

Yup